app/containers/MessageBox/UploadModal.js

```diff
@@ -63,18 +67,46 @@ const styles = StyleSheet.create({
 	androidButtonText: {
 		fontSize: 18,
 		textAlign: 'center'
+	},
+	fileIcon: {
+		color: COLOR_PRIMARY,
+		margin: 20,
+		flex: 1,
+		textAlign: 'center'
+	},
+	errorIcon: {
+		color: COLOR_DANGER
+	},
+	fileMime: {
+		color: COLOR_TEXT,
+		textAlign: 'center',
+		fontSize: 20,
+		marginBottom: 20
+	},
+	errorContainer: {
+		margin: 20,
+		flex: 1,
+		textAlign: 'center',
+		justifyContent: 'center',
+		alignItems: 'center'
 	}
 
 });
 
 @responsive
+@connect(state => ({
+	FileUpload_MediaTypeWhiteList: state.settings.FileUpload_MediaTypeWhiteList,
+	FileUpload_MaxFileSize: state.settings.FileUpload_MaxFileSize
+}))
 export default class UploadModal extends Component {
 	static propTypes = {
 		isVisible: PropTypes.bool,
 		file: PropTypes.object,
 		close: PropTypes.func,
 		submit: PropTypes.func,
-		window: PropTypes.object
+		window: PropTypes.object,
+		FileUpload_MediaTypeWhiteList: PropTypes.array,
```

Comment:
Expected array but this value is a String.

Reply:
yes you are right : thansk:)